src/features/Endpoint/Endpoint.module.scss

```diff
@@ -33,68 +33,89 @@
         input[type='number'] {
           background: 0 0;
           box-sizing: border-box;
-          color: var(--ifm-color-secondary-darkest);
-          display: block;
           height: rem(4);
           min-width: 0;
           width: 100%;
           border: none;
           text-indent: rem(1.2);
           &:not(:placeholder-shown) ~ label {
-            color: var(--ifm-color-secondary-darkest);
-            background-size: rem(1);
+            color: var(--colors-blue400);
+            background-color: var(--ifm-color-emphasis-0);
             padding: 0 rem(0.4);
             transform: translateY(rem(-2)) scale(0.75);
           }
-          &:focus {
-            outline-color: none;
-            & ~ label {
-              color: var(--ifm-color-secondary-darkest);
-              padding: 0 rem(0.4);
-              transform: translateY(rem(-2)) scale(0.75);
-            }
-          }
         }
         .inlineLabel {
-          position: relative;
+          position: absolute;
           display: inline-block;
-          font-size: var(--text-size-s);
-          margin-right: rem(0.5);
+          font-size: var(--fontSizes-xs);
           line-height: var(--text-lh-xl);
-          color: var(--black-01);
+          color: var(--colors-greyLight600);
           padding: rem(0.2) rem(0.4);
-          background-color: var(--ifm-color-secondary-darkest);
+          background-color: var(--ifm-background-surface-color);
           pointer-events: none;
           white-space: nowrap;
-          transform: translate(rem(0.6), rem(0.8));
+          left: rem(1);
+          top: rem(-0.8);
         }
         .textInput {
           background: none;
-          border: 1px solid var(--ifm-color-secondary-darkest);
+          border: 1px solid var(--colors-greyLight400);
+          border-radius: 4px;
           color: var(--ifm-font-color-base);
-          padding: rem(1);
+          padding: rem(1.2);
           width: 100%;
           font-size: var(--text-size-s);
+          &:focus {
+            border-color: var(--colors-blue400);
+            outline: none;
+            + .inlineLabel {
+              color: var(--colors-blue400);
+            }
+          }
+          &:hover {
+            border: 1px solid var(--colors-greyLight600);
+          }
         }
         .errorMessage {
+          position: relative;
           color: var(--ifm-color-danger);
-          font-size: var(--text-size-xs);
+          font-size: var(--fontSizes-2xs);
           font-weight: 400;
-          padding-left: rem(1);
+          padding: rem(0.5) rem(1);
         }
       }
     }
+    .currentUrl {
+      display: flex;
+      flex-direction: row;
+      font-size: var(--text-size-sm);
+      padding: 1rem;
+
+      .urlLabel {
+        font-weight: 700;
+        font-size: var(--text-size-sm);
+      }
+      .urlLink {
+        border-radius: 4px;
+        padding: 0.2rem 0.4rem;
+        color: #d91d4c; //var(--wine-01);
```

Comment:
Aah, I see that you used the direct color values from the current website.

It is best to replace this with one of our color variables (I suggest have a look at 'coral' and choose one of those colors).

Reply:
Alright!